src/LDPCDecoders.jl

```diff
@@ -6,6 +6,7 @@ using Random
 using SparseArrays
 using Statistics
 
+using QuantumClifford
```

Comment:
We should not depend on QuantumClifford, at least for now

However, we can totally depend on it in tests